src/usr/local/www/system_usermanager_addprivs.php

```diff
@@ -245,7 +245,7 @@ function get_root_priv_item_text() {
 			}
 			$desc = preg_replace("/pfSense/i", $g['product_name'], $pdata['descr']);
 			if (isset($pdata['warn']) && ($pdata['warn'] == 'standard-warning-root')) {
-				$desc .= ' ' . gettext('(This privilege effectively gives root privilege to the user)');
+				$desc .= ' ' . gettext('(This privilege effectively gives administrator-level privilege to the user)');
```

Comment:
s/privilege/access/ in this case to avoid awkward repetition of "privileges" in the sentence.